src/KM_Game.pas

```diff
@@ -740,6 +753,12 @@ procedure TKMGame.RequestGameHold(Msg: TGameResultMsg);
 
 procedure TKMGame.PlayerVictory(aPlayerIndex: TKMHandIndex);
 begin
+  fNetworking.PostLocalMessage(Format('%s has won!', //Todo translate
+    [fNetworking.GetNetPlayerByHandIndex(aPlayerIndex).NiknameColoredU]), csSystem);
```

Comment:
Today we discovered that this line crashes the game when a player is victorious in SP.
Logic should check if fGameMode = gmMulti ( Could just move it down a little. ;) )

Reply:
I added 
```if IsMultiplayer then```
in my repository there. 
Moving down a little will cause not showing win message for spectators, so its better leave it here. Or refactor all of the method (for me its ok, not a big deal)

Reply:
Both sound OK to me, though I think @Kromster80 and @lewinjh might have some opinions on this.